### PR TITLE
merge seaclouds-distribution and brooklyn-modaclouds capabilities

### PR DESCRIPTION
**Pull request by andreaturli**
add installer maven module
simplify the SeaClouds platform deployment on BYON and IaaS

we need volunteers to test the instructions on README. @SeaCloudsEU/developers anyone?

### CODE REVIEW

File: usage/installer/pom.xml [hunk 1]
```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2014 SeaClouds
+ 	Contact: Dionysis Athanasopoulos <dionysiscsuoi@gmail.com>
```
Comment (mbarrientos) [inline usage/installer/pom.xml]: I guess we can remove this contact address

File: usage/installer/src/main/assembly/files/conf/logback.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2014 SeaClouds
+ 	Contact: Dionysis Athanasopoulos <dionysiscsuoi@gmail.com>
```
Comment (mbarrientos) [inline usage/installer/src/main/assembly/files/conf/logback.xml]: Same here.

File: usage/installer/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Deploy SeaClouds
+A deployment of SeaClouds can be launched using Apache Brooklyn. We currently support deployments against Bring Your 
+Own Nodes (BYON) and to all the IaaS provider supported by [Apache jclouds](http://jclouds.org).
+
+# Deploy SeaClouds platform
+
+## Launching SeaClouds on the cloud
+
+```bash
+cd installer/target/seaclouds-installer-dist/seaclouds-installer
```
Comment (mbarrientos) [inline usage/installer/README.md]: `mvn clean install` is required before (even if obvious!) 
Also, the path should be `cd usage/installer/target/seaclouds-installer-dist/seaclouds-installer`.

File: usage/installer/src/main/assembly/files/conf/logback.xml (hunk 2 above)
Comment (andreaturli) [inline usage/installer/src/main/assembly/files/conf/logback.xml]: k

File: usage/installer/README.md (hunk 3 above)
Comment (andreaturli) [inline usage/installer/README.md]: ops, thx

Comment (andreaturli) [inline usage/installer/README.md]: I think this readme is more a developer guide as you need to build it from source

We need a simpler document that uses the binaries as well

File: pom.xml [hunk 4]
```diff
@@ -174,6 +176,10 @@
                         <exclude>**/*.json</exclude>
                         <exclude>**/*.yml</exclude>
                         <exclude>**/*.yaml</exclude>
+                        <exclude>**/*.sh</exclude>
+                        <exclude>**/Vagrantfile</exclude>
+                        <exclude>**/seaclouds_id_rsa</exclude>
+                        <exclude>**/seaclouds_id_rsa.pub</exclude>
```
Comment (mbarrientos) [inline pom.xml]: @andreaturli I found something strange. First `mvn clean install` fails on rat-check on this files, but a second run passes successfully. 

TravisCI also noticed it (line 1221): https://travis-ci.org/SeaCloudsEU/SeaCloudsPlatform/builds/65542052#L1221

Any ideas why this happens?

**Comment by mbarrientos** [comment]
Great PR, all the distribution efforts take a nice shape here. +1

File: usage/installer/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+pushd .
+cd byon
+vagrant up 
+popd
+nohup ./start.sh &
+tail -f nohup.out
+```
+This spins up a virtual environment, made up of 2 VMs, that is accessible at `192.168.100.10` and `192.168.100.11`.
+
+Finally, copy and paste [seaclouds blueprint](seaclouds-on-byon.yaml) to deploy the SeaClouds platform on the 2 VMs.
```
Comment (mbarrientos) [inline usage/installer/README.md]: This link seems broken. Maybe:
`./src/main/assembly/files/blueprints/seaclouds-on-byon.yaml`

Comment (andreaturli) [inline usage/installer/README.md]: thx

File: pom.xml (hunk 4 above)
Comment (andreaturli) [inline pom.xml]: mmm don't know why, I'll keep that in mind

**Comment by andreaturli** [comment]
@mbarrientos I'll rebase it with your comments addressed now. Let's see if travis is happy with the updated PR

**Comment by mbarrientos** [comment]
Thanks @andreaturli, merging.

**Pull request merged**